### ISSUE TITLE
assign_mtwcs fails when given MultiSlitModel data as input

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#5717]
_Issue [JP-1907](https://jira.stsci.edu/browse/JP-1907) was created on JIRA by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse):_

NIRSpec FixedSlit observations in Moving Target mode were conducted in nOPS10, and processing of those data crashed in the assign_mtwcs step with:
```java
2021-02-02 14:19:28,255 - stpipe.Spec3Pipeline.assign_mtwcs - INFO - Step assign_mtwcs parameters are: {'pre_hooks': [], 'post_hooks': [], 'output_file': None, 'output_dir': None, 'output_ext': '.fits', 'output_use_model': True, 'output_use_index': True, 'save_results': False, 'skip': False, 'suffix': 'assign_mtwcs', 'search_output_file': True, 'input_dir': ''}
----------------------------------------------------------------------
ERROR RUNNING STEP 'Spec3Pipeline':
    No attribute 'wcs'
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/bushouse/anaconda3/envs/jwstdev/lib/python3.9/site-packages/stdatamodels/properties.py", line 287, in __getattr__
    val = self._instance[attr]
KeyError: 'wcs'


During handling of the above exception, another exception occurred:


Traceback (most recent call last):
  File "/Users/bushouse/JWST/softw/jwst/scripts/strun", line 26, in <module>
    step = Step.from_cmdline(sys.argv[1:])
  File "/Users/bushouse/JWST/softw/jwst/jwst/stpipe/step.py", line 164, in from_cmdline
    return cmdline.step_from_cmdline(args)
  File "/Users/bushouse/JWST/softw/jwst/jwst/stpipe/cmdline.py", line 346, in step_from_cmdline
    step.run(*positional)
  File "/Users/bushouse/JWST/softw/jwst/jwst/stpipe/step.py", line 400, in run
    step_result = self.process(*args)
  File "/Users/bushouse/JWST/softw/jwst/jwst/pipeline/calwebb_spec3.py", line 118, in process
    input_models = self.assign_mtwcs(input_models)
  File "/Users/bushouse/JWST/softw/jwst/jwst/stpipe/step.py", line 400, in run
    step_result = self.process(*args)
  File "/Users/bushouse/JWST/softw/jwst/jwst/assign_mtwcs/assign_mtwcs_step.py", line 41, in process
    result = assign_moving_target_wcs(input)
  File "/Users/bushouse/JWST/softw/jwst/jwst/assign_mtwcs/moving_target_wcs.py", line 45, in assign_moving_target_wcs
    pipeline = model.meta.wcs._pipeline[:-1]
  File "/Users/bushouse/anaconda3/envs/jwstdev/lib/python3.9/site-packages/stdatamodels/properties.py", line 290, in __getattr__
    raise AttributeError("No attribute '{0}'".format(attr))
AttributeError: No attribute 'wcs'
```
For this particular dataset, this is due to the fact that the NRS FS 2D extracted data are stored in a MultiSlitModel (containing 1 slit) and hence attribute model.meta.wcs does not exist. Instead the correct reference is model.slits[0].meta.wcs.

The assign_mtwcs step needs to be updated to handle datamodels of this type.

**Comment by stscijgbot-jp** [comment]
_Comment by [Nadia Dencheva](https://jira.stsci.edu/secure/ViewProfile.jspa?name=dencheva) on [JIRA](https://jira.stsci.edu/browse/JP-1907?focusedCommentId=510273#comment-510273):_

I started work on it and moved it to "In Progress" on the periwinkle board. Not sure why it was marked as "In Testing".  Corrected this now. I think I've seen this behavior once before. Something to keep an eye on and correct if necessary.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-1907?focusedCommentId=510600#comment-510600):_

I've confirmed that the calwebb_spec3 now runs successfully on the original example data listed above. Closing.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-1907?focusedCommentId=510601#comment-510601):_

Fixed by #5758
